internal/pkg/fleetapi/client/client_test.go

```diff
@@ -183,6 +183,37 @@ func TestElasticApiVersion(t *testing.T) {
 		}
 	})
 
+	t.Run("verify that we don't log a generic 400 status as a downgrade request", func(t *testing.T) {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		mux := http.NewServeMux()
+		mux.HandleFunc("/someotheroperationreturnsbadrequest", func(writer http.ResponseWriter, request *http.Request) {
```

Comment:
[Suggestion]
Please could you at least use camelCase. It's hard to read the path as it's is and ensure the call uses the same path.

Reply:
I will put in the same url like the others

Reply:
done in 279b5275531a1932333e0d523e07d34c3cc38ba7